### PR TITLE
Introduction To Extended Reality

### PR DESCRIPTION
**Pull request by mbaabuones**
This is my first article as a contributor. My article is on 'Introduction to Extended Reality'. I have uploaded a file named 'index.md' and four other image files (virtual reality, augmented reality, xr games and virtual reality in medicine). I have met the writing guidelines that have been outlined.

### CODE REVIEW

**Comment by hectorkambow** [comment]
plagiarism check done 👍

File: articles/introduction-to-extended-reality/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Introduction to Extended Reality (XR)
+
+## What is Extended Reality (XR)?
+Extended reality (XR) is a term that encompasses real and virtual environments that are generated by wearable devices or computer technology to provide an immersive experience. It can also be described as a collection of all immersive technologies that combine real and virtual worlds. XR is growing in terms of advancement and application. This technology is still evolving because of ongoing development in human-machine interactions. 
+
+## Pillars of Extended Reality
+Extended reality encompasses three main pillars of immersive technology: Virtual Reality (VR), Augmented Reality (AR) and Mixed Reality (MR).
```
Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:7]: Augmented Reality (AR) and Mixed -> Augmented Reality (AR), and Mixed

File: articles/introduction-to-extended-reality/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Introduction to Extended Reality (XR)
+
+## What is Extended Reality (XR)?
```
Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:3]: Use H3(###) and beyond headings.

File: articles/introduction-to-extended-reality/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+![Augmented Reality](/engineering-education/introduction-to-extended-reality/augmented-reality.jpeg)
+
+[Image Source: Miro](https://miro.medium.com/max/1200/1*3F8SgP5Im17Kq9yPvy5rpA.jpeg)
+
+### Mixed Reality (MR)
+Mixed Reality encompasses the co-existence of real-world and digital objects. There is an interaction of these objects in real time. This is an advanced immersive technology that needs an MR headset and higher processing power than virtual reality and augmented reality. A good example of mixed reality is Microsoft's HoloLens, which allows users to interact with digital objects that are placed in a room.
+
+## How Extended Reality Works
```
Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:25]: H3 Heading needs to be used

File: articles/introduction-to-extended-reality/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+From a developer perspective, the extended reality is achieved by designing immersive logarithms. Software developers and data scientists should design clever and efficient algorithms to produce interesting XR features that offer unique user experiences. Other important concepts that need to be incorporated by developers of immersive technologies include 3D modelling and advanced computer vision. Artificially intelligent programs are made using machine learning, biomechanical modelling and motion tracking. Some of the programming languages used in the design of extended reality include C, C++, Java, JavaScript, Python and Swift. Full description of these languages and their application to AR can be found [here](https://howtocreateapps.com/programming-languages-ar/). Developers that may be interested in applying these programming languages in the development of VR can get more information [here](https://engineering.eckovation.com/programming-languages-develop-virtual-reality-apps/#:~:text=But%20here%20are%20four%20programming%20language%20you%20can,it%20has%20a%20huge%20...%20More%20items...%20).
```
Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:26]: What are immersive logarithms? Do you mean algorithms?

Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:26]: Divide this into two paragraphs. The second para should start from-> some of the programming languages...

File: articles/introduction-to-extended-reality/index.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  
+- __Hearing:__ Sounds in XR devices can be heard from all directions. These devices enable users to perceive sound to be in real situations. In football matches of video games, users will hear the sound of football fans, which happens in real life. 
+  
+- __Touch:__ The sense of touch is integrated into extended reality to improve user experience. In XR devices, gloves and bodysuits are used to give users a sense of touch within the simulation. Wearable devices use haptic technology, which employs a feedback system that helps in the expression of vibrations and motions.
+  
+- __Taste and Smell:__ These are new fields in extended reality that are undergoing research and development. Scent cartridges are used to provide a sense of smell. These cartridges produce pleasant, neutral and unpleasant scents. Electrodes can be placed on a user's tongue to provide a sense of taste.
+
+## Applications of Extended Reality
```
Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:39]: Use H3 here, and H4 and H5 for other sub-headings.

File: articles/introduction-to-extended-reality/index.md (hunk 3 above)
Comment (lalith1403) [inline articles/introduction-to-extended-reality/index.md:25]: Also, can you mention how AR, VR, and MR work to enable XR? What are their respective roles?

**Review by lalith1403 — CHANGES_REQUESTED** [review]
@mbaabuones Amazing article. I see this is your first article. You do a great job of explaining extended reality and its applications. I have a few suggestions that will enhance the readability of the article. 

Also, you will need to upload a short bio and a headshot image for your author page. Refer to the contributing guidelines for more details, or check the author's folder in the forked repository. Excited to read the updated article.

**Comment by mbaabuones** [comment]
@lalith1403 I appreciate your comments. I will work on these changes. Thank you.

File: articles/introduction-to-extended-reality/index.md (hunk 1 above)
Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: Done

File: articles/introduction-to-extended-reality/index.md (hunk 2 above)
Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: Done

File: articles/introduction-to-extended-reality/index.md (hunk 3 above)
Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: Done

File: articles/introduction-to-extended-reality/index.md (hunk 4 above)
Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: I meant immersive algorithms. I have made changes appropriately.

Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: Done

File: articles/introduction-to-extended-reality/index.md (hunk 5 above)
Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: Done

File: articles/introduction-to-extended-reality/index.md (hunk 3 above)
Comment (mbaabuones) [reply articles/introduction-to-extended-reality/index.md]: I have added a paragraph to incorporate this suggestion.

**Comment by mbaabuones** [comment]
@lalith1403 I have made all the requested changes. I have also uploaded a short bio and an image for my author page. Please check. Thank you.

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
@mbaabuones  great job - your article is getting ready to be published - keep up the great work 🚀

**Comment by mbaabuones** [comment]
@hectorkambow Thank you.

**Pull request merged**